app/middleware/renderer.js

```diff
@@ -1,8 +1,8 @@
 const log = require('../lib/logger');
 const mapLink = require('../lib/mapLink');
 
-function notFound(req, res, msg) {
-  log.warn(msg, 'Not found (404)');
+function notFound(req, res) {
+  log.warn({ req }, 'Not found (404)');
```

Comment:
I assume there's nothing of consequence in the message that's been removed, it'll say something like 'page not found'?